pkg/cmd/gist/view/view.go

```diff
@@ -38,7 +39,7 @@ func NewCmdView(f *cmdutil.Factory, runF func(*ViewOptions) error) *cobra.Comman
 		Args:  cmdutil.ExactArgs(1, "cannot view: gist argument required"),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			opts.Selector = args[0]
-
+			opts.IsPhysicalFile = false
```

Comment:
Is this field ever going to be true? If not, why would we need it?

Reply:
Same as above.